### ISSUE TITLE
Week 4

### ISSUE THREAD

**Issue by JoshuaW03628** [JoshuaW03628/Repository-1#6]
[Video](https://youtu.be/Nfy9keUAt-0)

**Comment by CodeingChris** [comment]
2.5 had no audio or edits, but everything else was good.

**Comment by drewreed2005** [comment]
Shows necessary parts with making changes, but video should be edited down. It is difficult to follow without sound. 2.6

**Comment by DreamerBlock** [comment]
no audio and is in very low fps and no edits but shows all thats needed

**Comment by jagermi3ster** [comment]
2.6 Video showed flask updating as well as fastpages running on a local server, video had no edits and no sound

**Comment by Toby-Leeder** [comment]
No sound, hard to tell what's happening. Some weird screen glitches, showed all requirements except github change.

**Comment by Nathan-Capule** [comment]
No sound and did not talk in person. But was able to show local changes, also did not show collaboration with teammates.

**Comment by GeneChang1** [comment]
No sound, kind of glitchy. Could have edited the mistakes out, don't need to commit changes to see it in local server. Didn't see github changes.

**Comment by trentcardall** [comment]
2.6

Shows servers running and changes being made but no sound or edits

**Comment by jm1021** [comment]
No sound in video.  Hard to follow.  No captions. Shows key steps, but not sure about transition to live server.   2.4

**Comment by TheGerbil21** [comment]
Really wish there was sound and better video quality, it looked like you had some of the requirements done. 2.5

**Comment by CubNavarro** [comment]
had no sound and showed everything that was needed and had the requirements. 2.5
